app/javascript/bundles/UI/components/Alert/Alert.scss

```diff
@@ -7,23 +7,19 @@
   color: #000000;
 
   &.alert-standard {
-    background-color: #ffffff;
-    border-color: #f2f2f2;
-  }
-  &.alert-sky {
     color: #ffffff;
     border-color: #b8daff;
     background-color: #354093;
   }
-  &.alert-fire {
+  &.alert-error {
     border-color: #b8daff;
```

Comment:
Also, if you take a closer look into the Figma document you will see that only the default alert has a border. Please, check out your css to match the Figma specification :wink: 
https://www.figma.com/file/PuezufmBksh475JS5nvBsSkx/ConexionFelina?node-id=0%3A1